datanode/src/storage_api_test.py

```diff
@@ -101,35 +101,35 @@ def testSlippyConversionWithValidData(self):
     r = self.app.get('/slippy/11?coords=1,1')
     self.assertEqual(r.status_code, 200)
     j = json.loads(r.data)
-    self.assertEqual(j['data']['tiles'][0], [11, 1029, 1018])
-    self.assertEqual(j['data']['links'][0],
-                    'http://tile.openstreetmap.org/11/1029/1018.png')
-    r = self.app.get('/slippy/10?coords=37.203335,-80.599481')
-    r = self.app.get('/slippy/10?coords=37.203335,-80.599481')
+    self.assertEqual(11, j['data']['grid_cells'][0]['zoom'], 11)
```

Comment:
I think we want better assert failure messages here (third argument to assertEqual), or omit the third argument.

Reply:
Roger. Typo.